### PR TITLE
Pull Dev -> master 

### PR DESCRIPTION
**Pull request by ramdootp**
Pull Dev -> master

### CODE REVIEW

**Comment by amsaha** [comment]
@swiftdiaries can you also take a look since this is a big merge?

**Comment by swiftdiaries** [comment]
Ok 🙌🏾

File: apps/healthcare/covid-forecasting/onprem/fairing/COVID-Forecast-fairing.ipynb [hunk 1]
```diff
@@ -0,0 +1,3216 @@
+{
```
Comment (swiftdiaries) [inline apps/healthcare/covid-forecasting/onprem/fairing/COVID-Forecast-fairing.ipynb:1]: <p>This is nice :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (swiftdiaries) [inline apps/healthcare/covid-forecasting/onprem/fairing/COVID-Forecast-fairing.ipynb:1]: <p>Can we move the <code>DOCKER_REGISTRY</code>  to the top cell?</p><p>Or reuse the <code>$USERNAME</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: apps/healthcare/covid-forecasting/onprem/pipelines/components/covid_preprocess/build_image.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+   exit 1
+fi
+
+if [ -z "${TAG_NAME}" ]; then
+  TAG_NAME=$(date +v%Y%m%d)-$(git describe --tags --always --dirty)-$(git diff | shasum -a256 | cut -c -6)
+fi
+
+docker build -t ${USER_NAME}/${LOCAL_IMAGE_NAME}:${TAG_NAME} .
+if [ -z "${IMAGE_NAME}" ]; then
+  cat ~/my_password.txt | docker login --username ${USER_NAME} --password-stdin
```
Comment (swiftdiaries) [inline apps/healthcare/covid-forecasting/onprem/pipelines/components/covid_preprocess/build_image.sh:54]: This is cool :)

**Pull request merged**